src/main/java/com/example/demo/controller/MailSenderController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.Message;
+import com.example.demo.repo.JsonRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController()
+@RequestMapping("/message")
```

Comment:
As we've discussed on a lecture when you want to address to collection of objects you need to use plural. Please, update this common path to */messages*

Reply:
Sorry, I'll fix it